react-jsx/package.json

```diff
@@ -11,10 +11,11 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@babel/core": "^7.16.12",
+    "@babel/plugin-transform-react-jsx": "^7.16.7",
+    "babel-loader": "^8.2.3",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
-  },
-  "devDependencies": {
+    "react-dom": "^17.0.2",
     "webpack": "^5.68.0",
     "webpack-cli": "^4.9.2"
```

Comment:
All webpack and babel packages should be under `devDependencies`, `react` and `react-dom` should be under `dependencies`